Module resolution fails when used Path mapping

I was trying to use path mapping feature https://www.typescriptlang.org/docs/handbook/module-resolution.html . I see source compiling  with 'tsc' , However when I used grunt-ts to compile same source , it fails with module not found error ("typescript": "2.4.2", "grunt-ts": "^6.0.0-beta.17",)

Using tsc v2.4.2
src/js/utils/globalConfig.ts(7,36): error TS2307: Cannot find module '@SomeModel'.

Is this a bug or not supported case?

--- a/README.md
+++ b/README.md
@@ -88,6 +88,7 @@ A more extensive sample `Gruntfile.js` is available [here](https://github.com/Ty
  * [Encodes HTML](#html) files as TypeScript variables (for HTML templating engines)
  * Performs live file [watching](#watch) (compile on save)
  * Enables ["Fast" compile](#fast) when using external modules
+ * Please not that Grunt-ts does not support the [Path Mapping](https://www.typescriptlang.org/docs/handbook/module-resolution.html#path-mapping) feature as it is not supported by `tsc` from the command-line
 
 ## Support for tsc Switches
 

